tests/fixtures/my.cnf.tmpl

```diff
@@ -7,12 +7,12 @@ port = {port}
 host = {host}
 password = {password}
 database = {db}
-socket = /var/run/mysqld/mysqld.sock
+socket = /tmp/aiomysql/mysqld.sock
```

Comment:
This socket path and the following ones are taken from [docker-compose volume config](https://github.com/aio-libs/aiomysql/blob/master/docker-compose.yml#L11).